### PR TITLE
Hotfix/using sprockets for static applications

### PR DESCRIPTION
**Pull request by bwl21**
As discussed on Gitter, I have updated static_applications.html.md with respect to opal 0.8.0.

### CODE REVIEW

File: source/docs/static_applications.html.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ First, install `opal` adding them to a `Gemfile`:
 
 ```ruby
 # Gemfile
-gem "opal", "~> 0.7.0"
+
+gem "opal", ">= 0.8.0"
```
Comment (bbatsov) [inline source/docs/static_applications.html.md]: Why did you add blank lines around this line?

File: source/docs/static_applications.html.md [hunk 2]
```diff
@@ -28,10 +30,16 @@ You will notice the `require "opal"` line which will automatically include the
 opal runtime and corelib into our output, giving us access to the `puts()`
 method.
 
-To build this, we need the rake task to add our
-`app/` directory to the load path, and then to build our target file
+To build this, we can use a rake task to add our
+`app/` directory to the load path, and then to build our target file `build.js` from
 `application.rb` which will be found because it is inside our added load path.
 
+If you want to use gems installed on your build machine, tell Opal such that Opal can find it
```
Comment (bbatsov) [inline source/docs/static_applications.html.md]: such that -> so that

File: source/docs/static_applications.html.md [hunk 3]
```diff
@@ -28,10 +30,16 @@ You will notice the `require "opal"` line which will automatically include the
...
+(e.g. `Opal.use_gem "faker"`). If you require a gem it will be required during the compilation 
+process and will be included into the generated JavaScript. 
+
+
```
Comment (bbatsov) [inline source/docs/static_applications.html.md]: You should kill the final couple of blanks.

File: source/docs/static_applications.html.md [hunk 4]
```diff
@@ -39,10 +47,16 @@ require 'opal'
...
   Opal.append_path "app"
+  Opal.use_gem "faker"
+  
   File.binwrite "build.js", Opal::Builder.build("application").to_s
 end
 ```
 
+
+
+```ruby
```
Comment (bbatsov) [inline source/docs/static_applications.html.md]: This looks like some mistake. Starting a code block in the middle of nowhere.

File: source/docs/static_applications.html.md [hunk 5]
```diff
@@ -61,3 +75,33 @@ To run the application, lets create a very simple html file:
 
 Now, open this html file and check the browsers console. You should see our
 message printed in the console.
+
+
+Note that you can also use sprockets to build the target if you want to.
+
+Dependency management is delegated directly to Sprockets.
```
Comment (bbatsov) [inline source/docs/static_applications.html.md]: sprockets

**Comment by bbatsov** [comment]
You should squash your commits together.

File: source/docs/static_applications.html.md (hunk 1 above)
Comment (bwl21) [inline source/docs/static_applications.html.md]: it is a matter of taste. I removed it.

**Comment by bwl21** [comment]
Even if we do not yet have a dedicated use case which justifies sprockets for static applications, I still think, it might be worth to keep this in the documentation as we discussed on Gittet at 18.July. It at least helps to understand the whole thing more.
